client/src/components/pages/MembersPage/components/CreateForm/CreateForm.js

```diff
@@ -0,0 +1,186 @@
+import React, { useRef } from 'react'
+import {
+  Form,
+  Row,
+  RowFlexStart,
+  CheckboxRow,
+  CheckboxDescription,
+  Column,
+  FormTitle,
+  InputField,
+  BandSelector,
+  BandSelectOption,
+  Tag,
+  AddNewMembersInstrument,
+  CloseForm,
+  Submit,
+} from './CreateFormStyledComponents.js'
+
+function CreateForm({
+  members,
+  setMembers,
+  postMember,
+  newMembersInstruments,
+  addNewMembersInstrument,
+  removeNewMembersInstrument,
+  resetNewMembersInstruments,
+  bands,
+  setBands,
+  toggleForm,
+}) {
+  const newInstrumentInputRef = useRef()
+
+  return (
+    <Form onSubmit={createMember}>
+      <CloseForm onClick={toggleForm}>
+        <i className="fas fa-times-circle" style={{ background: '0' }}></i>
+      </CloseForm>
+      <FormTitle>
+        <h3>Create Form</h3>
+      </FormTitle>
+
+      <InputField
+        type="text"
+        name="memberName"
+        placeholder="New member's name"
+        autoFocus
+      />
+      <Column>
+        <BandSelector name="band">
+          {bands.map(band => (
+            <BandSelectOption value={band.value} key={band.value}>
+              {band.name}
+            </BandSelectOption>
+          ))}
+        </BandSelector>
+
+        <InputField type="text" name="newBand" placeholder="new band" />
+      </Column>
+      <Row>
```

Comment:
Row in column in row?
This seems needlessly complicated for what the form actually looks like, maybe rethink with a simple flex column and the occasional flex row sprinkled in?